### ISSUE TITLE
Add info/content on the WCAG exemptions in regulatory work

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#145]
>Both [US Section 508](https://www.access-board.gov/ict/#b-application-of-wcag-2-0-to-non-web-ict) and [EN 301 549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf) have exemptions to applying four WCAG 2.0 SC:
>* [2.4.1 Bypass Blocks](https://www.w3.org/TR/wcag2ict/#navigation-mechanisms-skip)
>* [2.4.5 Multiple Ways](https://www.w3.org/TR/wcag2ict/#navigation-mechanisms-mult-loc)
>* [3.2.3 Consistent Navigation](https://www.w3.org/TR/wcag2ict/#consistent-behavior-consistent-locations)
>* [3.2.4 Consistent Identification](https://www.w3.org/TR/wcag2ict/#consistent-behavior-consistent-functionality) 
>
>These exemptions should also be added to the updated WCAG2ICT. 

Quoted from [Discussion topic from Sam](https://github.com/w3c/wcag2ict/discussions/50)

**Comment by maryjom** [comment]
[Survey results](https://www.w3.org/2002/09/wbs/55145/WCAG2ICT-public-comment-group2/results#xq2) polling the TF's thoughts on this.  Follow-up [discussion in 22 Feb. TF meeting](https://www.w3.org/2024/02/22-wcag2ict-minutes#t09), though brief.

**Comment by maryjom** [comment]
In the survey, Loïc suggested the following text, which is a good starting point:
>NOTE: This success criterion is being excepted in EN 301 549 for non-web documents and software programs because "set of non-web documents or software programs" are considered to be rare.

**Comment by maryjom** [comment]
Per the more extensive TF [discussion of the survey results on 29 February](https://www.w3.org/2024/02/29-wcag2ict-minutes#t03), the TF agreed we should include a more generic statement that indicates to look into the regulations and standards where WCAG is applied to non-web technologies to see how they handled the "sets of" SCs.  

My post-meeting thought: We may want to think about having a similar statement in the SC Problematic for Closed Functionality as the interpretation is different for 508 vs. the EN on this.

**Comment by maryjom** [comment]
@loicmn @Lboniello @bruce-usab How's the proposal going? How can I help?

**Comment by bruce-usab** [comment]
Follows is a first draft, only addressing U.S. 508.

The [2013 WG Note](https://www.w3.org/TR/2013/NOTE-wcag2ict-20130905/) impacted how WCAG 2.0 was adopted into U.S. 508 Regulation.  See preamble of final rule, [B. Application of WCAG 2.0 to Non-Web ICT](https://www.federalregister.gov/documents/2017/01/18/2017-00395/information-and-communication-technology-ict-standards-and-guidelines#h-36) (82 FR 5790, 18 Jan 2017).  This [section concludes](https://www.federalregister.gov/d/2017-00395/p-71) (emphasis added):
> To address concerns expressed by some commenters and the working group regarding the application of a few WCAG 2.0 Success Criteria to non-Web documents and non-Web software, in the final rule we have excepted non-Web documents and non-Web software from compliance with these criteria. Specifically, **non-Web documents and non-Web software need not comply with WCAG 2.0 Success Criteria 2.4.1 Bypass Blocks, 2.4.5 Multiple Ways, 3.2.3 Consistent Navigation, and 3.2.4 Consistent Identification**. Additionally, we added new provisions to instruct the reader when applying WCAG 2.0 to non-Web documents and non-Web software to replace the term “Web page” with the term “document” or “software.” We added this exception and new provisions where applicable throughout the final rule text.

**Comment by pday1** [comment]
Very rough notes from discussion on 15 March. Language needs polishing, but the idea is something along these lines is added to the end of the Background section. 

_Not all SCs have been fully adopted in all local regulations and legislation, and may not be applicable to all technologies. WCAG2ICT was also used to determine whether or not to apply certain SCs. For example, some local standards such as Section 508 in the US, non-Web documents and non-Web software need not comply with WCAG 2.0 Success Criteria 2.4.1 Bypass Blocks, 2.4.5 Multiple Ways, 3.2.3 Consistent Navigation, and 3.2.4 Consistent Identification. Regulators should consider the applicability of individual SCs._

**Comment by pday1** [comment]
Also add general sentence to each of the "sets of" SCs that points back to this new paragraph in the Background section. 

Local regulations and guidance may not apply this SC, see background section in document for more information

**Comment by loicmn** [comment]
Sorry for my lack of activity. I suggest to add some text to also refer to EN 301 549, expanding @pday1's proposal (I've also expanded SCs as success criteria):

> Not all success criteria have been fully adopted in all local regulations and legislation, and may not be applicable to all technologies. WCAG2ICT was also used to determine whether or not to apply certain success criteria. For example, some local standards such as Section 508 in the US, and EN 301 549 in Europe, state that non-Web documents and non-Web software do not need to comply with WCAG 2.0 Success Criteria 2.4.1 Bypass Blocks, 2.4.5 Multiple Ways, 3.2.3 Consistent Navigation, and 3.2.4 Consistent Identification. In addition, EN 301 549 also states that non-Web software does not need to comply with 2.4.2 Page titled and 3.1.2 Language of parts. Regulators should consider the applicability of individual success criteria.

**Comment by GreggVan** [comment]
I think everything up to the last sentence is informative.

this sentence is a big dangerous  and sounds like you mean shopping WCAG provisions for web content
>  Regulators should consider the applicability of individual success criteria.
> 
I guess it is ok — but you need to add to the end    …. to non-web content
so that it reads 

>  Regulators should consider the applicability of individual success criteria to non-web content.
> 


G

**Comment by maryjom** [comment]
As an FYI, there is already some content regarding Section 508 and EN 301 549 use of WCAG2ICT in the  [Background section](https://w3c.github.io/wcag2ict/#background) of the document. I cannot tell from the above proposals whether this content is intended to amend that section or to be added to the individual "Applying Success Criteria..." sections for the 5 SC that are interpreted to apply to "sets of non-web documents and software".  

My personal recollection from our meeting discussion, as documented in our meeting [minutes from 29 February](https://www.w3.org/2024/02/29-wcag2ict-minutes#t03) was that we would keep the statement quite simple.  See the poll at the end of the topic with where we agreed: 
> Take the direction that we generically say to look to regulatory standards that apply WCAG to see exactly if and how this SC is applied.

I suggest something like:
> NOTE: Check regulatory standards that apply WCAG to non-web documents and software (e.g., U.S. Revised 508 and EN 301 549 ) to determine whether or how this success criterion is applied.

**Comment by maryjom** [comment]
Any progress? Need to get a proposal together to survey so we can close this out.

**Comment by GreggVan** [comment]
PROPOSAL
NOTE: see special comment in section [name location in WCAG2ICT] titled **Treatment of SC with "set of web pages" in them**

**Treatment of SC with "set of web pages" in Them**
There are several WCAG Success Criteria where the phrase "set of web pages" is used.  These are meant to apply to sets of "pages" or "screens" that are meant to be treated as a set rather than as individual items.   
For documents - the closest thing would be the pages in a single document.  It could also be a set of documents meant to be treated as single unit that is published together at the same time.  Options may be to "apply as written to the pages in a single document" or "apply as writtento sets of documents that are published and always updated together at the same time and intended to be treated as a single item" such as a set of volumes.   
For software - the closest thing would be the different parts or views within a software program.  However, we found no way to define discreet "different parts or views" of software.  And sets of software that are meant to be used as a unit and always published and updated at the same time are rare.   Options for software may be to make this a should and word it as applying within software - or skipping this when applying this to software.

**Comment by pday1** [comment]
Latest from issue conversation: taken from Loic's input, and Gregg's comments on that.


Not all success criteria have been fully adopted in all local regulations and legislation, and may not be applicable to all technologies. WCAG2ICT was also used to determine whether or not to apply certain success criteria. For example, some local standards such as Section 508 in the US, and EN 301 549 in Europe, state that non-Web documents and non-Web software do not need to comply with WCAG 2.0 Success Criteria 2.4.1 Bypass Blocks, 2.4.5 Multiple Ways, 3.2.3 Consistent Navigation, and 3.2.4 Consistent Identification. In addition, EN 301 549 also states that non-Web software does not need to comply with 2.4.2 Page titled and 3.1.2 Language of parts. Regulators should consider the applicability of individual success criteria to non-web content.

**Comment by pday1** [comment]
Content to be inserted to end of Guidance section, with new sub-heading. (Possibly "Interpretation of web terminology in a non-web context"). 

Then this content is inserted after NOTE 2

> Not all success criteria have been fully adopted in all local regulations and legislation, and may not be applicable to all technologies. WCAG2ICT was also used to determine whether or not to apply certain success criteria. For example, some local standards such as Section 508 in the US, and EN 301 549 in Europe, state that non-Web documents and non-Web software do not need to comply with WCAG 2.0 Success Criteria 2.4.1 Bypass Blocks, 2.4.5 Multiple Ways, 3.2.3 Consistent Navigation, and 3.2.4 Consistent Identification. In addition, EN 301 549 also states that non-Web software does not need to comply with 2.4.2 Page titled and 3.1.2 Language of parts. Regulators should consider the applicability of individual success criteria to non-web documents and software.

**Comment by pday1** [comment]
Then for adding to the 5 SCs: Regulators should consider the applicability of individual success criteria to non-web documents and software. See section [ref to the new section at end of guidance section]

**Comment by maryjom** [comment]
PR #330 is ready for TF review.

**Comment by maryjom** [comment]
The TF began discussions of the [survey results reviewing PR 330](https://www.w3.org/2002/09/wbs/55145/WCAG2ICT-survey-comment-answers/results) on 18 April, but only got as far as the first question. (See [discussion of survey results](https://www.w3.org/2024/04/18-wcag2ict-minutes#t06)). Still mulling over what would be the details of the text needed in light of the new DoJ Rule which points implementers to W3C guidance.

**Comment by maryjom** [comment]
During the 2 May meeting, the TF reached consensus on the content which will be incorporated into PR #330 and merged into the document.
